package.json

```diff
@@ -17,8 +17,8 @@
     "url": "https://github.com/puleos/object-hash/issues"
   },
   "scripts": {
-    "test": "node ./node_modules/.bin/mocha test",
-    "prepublish": "gulp dist"
+    "test": "gulp test",
+    "prepublish": "rollup -c",
```

Comment:
I don’t know why Github doesn’t show the Travis CI results anymore, but npm is failing here:

https://travis-ci.org/github/puleos/object-hash/jobs/674017515

Reply:
Yep, invalid trailing comma.